AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -25,6 +25,7 @@
 1track.ru##.offer_app
 matchtv.ru##div[data-promo-desktop-url^="https://apps.rustore.ru/"]
 home.kingsoft.jp#?#.popup:has(button#open-app)
+pokemongo-get.com###text-26[class="widget-over-sns-buttons widget_text"]
```

Comment:
I can't check it with VPN. Could you provide the source of the element?
@Yuki2718 please also take a look.

Reply:
`pokemongo-get.com###text-26.widget-over-sns-buttons` is fine in this case.

Reply:
This is the source of that element.
<details><summary>screenshot</summary>

![](https://a.uguu.se/iZBetEzM.png)
</details><br/>